core/data/tests/use_correct_integer_types/output.py

```diff
@@ -0,0 +1,20 @@
+"""
+ Generated by typeshare 1.12.0
+"""
+from __future__ import annotations
+
+from pydantic import BaseModel
+
+
+class Foo(BaseModel):
+    """
+    This is a comment.
+    """
+    a: int
```

Comment:
Nit: I don't feel nearly as strongly about this as I do about by other feedback, but if you wanted to be fancy, you could use `conint` to handle fixed-width integers with a min and max value: https://docs.pydantic.dev/1.10/usage/types/#arguments-to-conint

Reply:
I can't resolve this thread for some reason. Marking as won-fix to prevent scope-creeping.